packages/client/components/EstimatePhaseDiscussionDrawer.tsx

```diff
@@ -32,6 +32,7 @@ const Drawer = styled('div')<{isDesktop: boolean; isOpen: boolean}>(({isDesktop,
   zIndex: ZIndex.SIDEBAR,
   [`@media screen and (max-width: ${Breakpoint.POKER_DISCUSSION_FULLSCREEN_DRAWER}px)`]: {
     width: '100vw',
+    height: '100dvh',
```

Comment:
@Dschoordsch
`dvh` currently does not work on chrome

Reply:
My fault, thanks for checking!

Reply:
@Dschoordsch  | @87prashant  whats the way forward on this.